releasenotes/notes/fix-deprecated-set-meta-set-metas-1b348cdbff41fc1a.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixes :py:meth:`ddtrace.Span.set_meta` and :py:meth:`ddtrace.Span.set_metas` deprecation warnings 
```

Comment:
Was this change released yet? If not then we don't need the release note

Reply:
Nope